style.css

```diff
@@ -0,0 +1,162 @@
+body {
+  background-color: lightgrey;
+}
+
+#first-header {
+  height: 105px;
+}
+
+.header-color {
+  background-color: #f72210;
+}
+
+.navbar-brand {
+  font-weight: 900;
+}
+
+.subscribe-btn {
+  background-color: #ffe400;
+  color: black;
+  border-radius: 24px;
+  font-weight: 800;
+  font-size: 100%;
+}
+
+a:link {
+  text-decoration: none;
+}
+
+.info-author {
+  font-size: 17px;
+  font-weight: 700;
+}
+
+.info-title {
+  font-size: 14px;
+  font-weight: 800;
+}
+
+.hr-txt-parent {
+  width: 100%;
+  text-align: center;
+  border-bottom: 1px solid rgb(219, 219, 219);
+  line-height: 0.1em;
+  margin: 20px 0;
+}
+
+.hr-txt {
+  padding: 0 10px;
+  border-radius: 50%;
+  color: aliceblue;
+  font-weight: 800;
+}
+
+/* Magazine */
+
+.font-weight-bold {
+  font-weight: bold;
+}
+
+/* Footer */
+.footer-link {
+  border-right: 1px solid #d9d9d9;
+  margin: 10px 0;
+}
+
+#footer-icons i {
+  color: #808080;
+  height: 20px;
+  width: 20px;
+}
+
+.image-contents {
+  position: relative;
+}
+
+.left-image-text {
+  position: absolute;
+  bottom: -0.5em;
+  left: 0;
+  padding: 0 0.2em;
+  background-color: whitesmoke;
+  opacity: 0.8;
+  color: orangered;
+  font-weight: 700;
+}
+
+.left-image-text-2 {
+  position: absolute;
+  bottom: -0.5em;
+  left: 0;
+  padding: 0 0.2em;
+  background-color: whitesmoke;
+  opacity: 0.8;
+  font-weight: 700;
+}
+
+.bold-link {
+  font-weight: 700;
+  font-size: 1.5em;
+}
+
+.editions {
+  flex-wrap: wrap;
+}
+
+.editions a {
+  font-size: 1em;
+}
+
+#two-img {
+  position: relative;
+}
+
+#img-1,
+#img-2 {
+  position: absolute;
+  height: 90%;
+  width: 70%;
+}
+
+#img-2 {
+  bottom: 0;
+  right: 0;
+}
+
+#img-1 {
+  left: 0;
+  top: 0;
+}
+
+@media (min-width: 992px) {
+  #navbar-md-sm {
+    display: none;
+  }
+}
+
+@media (max-width: 991px) {
+  #first-header,
+  #second-header,
+  #third-header {
+    display: none !important;
```

Comment:
- [ ] Remove all `!important` from the CSS stylesheet. don't override classes, and find a way whether with Bootstrap classes or with custom CSS is Bootstrap doesn't have a class for it.